src/Estimators/InputSection.h

```diff
@@ -61,7 +61,7 @@ class InputSection
   // Enable read-only access to variable values.
   //   Needs updating to allow copy-less return.
   template<typename T>
-  T get(const std::string& name) const
+  const T get(const std::string& name) const
```

Comment:
const has no effect when return by value.

Reply:
will fix in later PR